docs/BitcoinCoreKeys.md

```diff
@@ -0,0 +1,119 @@
+ eclair.signer.passphrase | passphrase                                                               
+ eclair.signer.timestamp  | wallet creation UNIX timestamp. Set to the current time for new wallets. 
+
+This is an example of `eclair-signer.conf` configuration file:
+
+```hocon
+{
+   eclair {
+      signer {
+         wallet = "eclair"
+         mnemonics = "legal winner thank year wave sausage worth useful legal winner thank year wave sausage worth useful legal winner thank year wave sausage worth title"
+         passphrase = ""
+         timestamp = 1686055705
+      }
+   }
+}
+```
+
+3) Configure Eclair to handle private keys for this wallet
+
+Set `eclair.bitcoind.wallet` to the name of the wallet just created (`eclair` in the example above) and restart Eclair.
+Eclair will automatically create a new, empty, descriptor-enabled, watch-only wallet in Bitcoin Core and import its descriptors.
+
+:warning: Eclair will not import descriptors if the timestamp set in your `eclair-signer.conf` is more than 2 hours old. If the mnmeonics and
+passphrase that your are using are new, you can safely update this timestamp, but if they have been used before you will need to follow
+the steps described in the next section.
+
+You now have a Bitcoin Core watch-only wallet for which only your Eclair node can sign transactions. This Bitcoin Core wallet can
+safely be copied to another Bitcoin Core node to monitor your onchain funds.
+
+You can also use `eclair-cli getmasterxpub` to get a BIP32 extended public key that you can import into any compatible Bitcoin wallet
+to create a watch-only wallet (Electrum for example) that you can use to monitor your Bitcoin Core balance.
+
+:warning: this means that your Bitcoin Core wallet cannot send funds on its on (since it cannot access private keys to sign transactions).
+To send funds onchain you must use `eclair-cli sendonchain`.
+
+:warning: to backup the private keys of this wallet you must either backup your mnemonic code and passphrase, or backup the `eclair-signer.conf` file in your eclair
+directory (default is `~/.eclair`) along with your channels and node seed files.
+
+:warning: You can also initialise a backup onchain wallet with the same mnemonic code and passphrase (a hardware wallet for example), but be warned that using them may interfere with your node's operations (for example you may end up
+double-spending funding transactions generated by your node).
+
+## Importing an existing Eclair-backed bitcoin core wallet
+
+The steps above describe how you can simply switch to a new Eclair-backed bitcoin wallet.
+If you are already using an Eclair-backed bitcoin wallet that you want to move to another setup, you have several options:
+
+### Copy the bitcoin core wallet and `eclair-signer.conf`
+
+Copy your wallet file to a new Bitcoin Core node, and load it with `bitcoin-cli loadwallet`. Bitcoin Core make need to scan the blockchain which could take some time.
```

Comment:
typo: Bitcoin Core make need to scan -> Bitcoin Core may need to scan

Reply:
fixed in https://github.com/ACINQ/eclair/pull/2613/commits/c00ae33f62d53d4d3710f7dc05a273993eb47657